OpenRA.Game/Activities/CallFunc.cs

```diff
@@ -29,5 +33,15 @@ public override Activity Tick(Actor self)
 			if (a != null) a();
 			return NextActivity;
 		}
+
+		public void SetTargetLineNode(Target target, Color color, bool isTerminal)
```

Comment:
Can we please default `bool isTerminal = false` so that most of the activities don't need to declare that?